### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Arcziwald**
Here's the proposal :-) I'm counting on your suggestions and tips so that the final result will be great :-)
We will put the GIF at the end :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
 - [x] [Krystian Żybura](https://github.com/KrystianZybura)
 - [x] [Artur Mochnia](https://github.com/Arcziwald)
+
+## Technologies:
+- React-Router
+- 
+- 
+- 
+- 
+-
```
Comment (KrystianZybura) [inline README.md]: We can make something like a complete list of technologies here and also maybe something called "Technologies shortcut", which would display some images with js, react, css icons etc.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,38 @@
-# Welcome on our Movies Browser site (... in progress ;)
+# Welcome on our Movies Browser
```
Comment (KrystianZybura) [inline README.md]: I'd like to add a word "Project" after the "Movies Browser"

**Review by KrystianZybura — COMMENTED** [review]
Hey, nice that you've brought this initial readme setup ✌. There are possible many improvements, we should consider using emojis, maybe some icons or even animations, for example animated readme title, or maybe some other fancy things 😎

File: README.md (hunk 1 above)
Comment (Arcziwald) [reply README.md]: create without limits :-)

File: README.md [hunk 3]
```diff
@@ -1,9 +1,45 @@
...
+## 📓 Description
+
+Our project is an advanced movie and actor search engine that utilizes a rich data source from The Movie Database. Our three-person team has been working on this project, using various technologies to create a comprehensive application for searching movies and actor information. We hope that our movie and actor search engine will be useful for all cinema enthusiasts. If you have any questions or feedback, please don't hesitate to contact us.
+
+## Main features:
+
+### Movies Search:
+
+Our app allows users to easily search for movies by title, genre, year of production and many other criteria.
+
```
Comment (WoytekMig) [inline README.md:34]: not so sure, if we allow user to search by genre or year ... :D

File: README.md [hunk 4]
```diff
@@ -1,9 +1,45 @@
-# Welcome on our Movies Browser site (... in progress ;)
+[![Typing SVG](https://readme-typing-svg.demolab.com?font=Fira+Code&size=28&pause=1000&color=B000FF&center=true&vCenter=true&width=650&lines=Welcome+on+our+Movies+Browser+Project)](https://git.io/typing-svg)
```
Comment (WoytekMig) [inline README.md:2]: wooah ! :D IT'S MOVING :D great :)

**Review by WoytekMig — APPROVED** [review]
I think it's attractive ! Good job :)

**Review by KrystianZybura — APPROVED** [review]

**Review by KrystianZybura — APPROVED** [review]

**Pull request merged**